### DISCUSSION TITLE
Authentication example /(app)/index route.pathname changed since 0.0.37

### DISCUSSION THREAD

**Discussion by cseelus** [expo/router#169] [Q&A] [answered]
With Expo Router ≤ 0.0.36, `route.pathname` (with `route = useHref()`) for `/(app)/index.js` from the [Authentication example](https://expo.github.io/router/docs/guides/auth) was just `/`.

Since upgrading, it is `/(app)`.

Don't know it this is a regression or a change, since I wasn't able to find the changelog for this project yet.

**Comment by EvanBacon** [comment, marked as answer]
Yes, to accommodate _Share Routes_ we now include the group segments in the pathname (renamed to `useSegments`). This enables the ability to distinguish if you have a matching route in a different group.
